compat/src/index.d.ts

```diff
@@ -394,6 +394,9 @@ declare namespace React {
 	export function flushSync<R>(fn: () => R): R;
 	export function flushSync<A, R>(fn: (a: A) => R, a: A): R;
 
+	export function unstable_batchedUpdates<A, R>(callback: (a: A) => R, a: A): R;
+	export function unstable_batchedUpdates<R>(callback: () => R): R;
```

Comment:
React's types, for comparison:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/47f29647038efc420674f61b1a3aaada32444640/types/react-dom/index.d.ts#L19-L20